docs/RFCS/client.Txn-TxnCoordSender_split.md

```diff
@@ -0,0 +1,175 @@
+- Feature Name: Eliminating the `client.Txn` ↔ `TxnCoordSender` split
+- Status: draft/in-progress/completed/rejected/obsolete
+- Start Date: 2017-05-17
+- Authors: Andrei Matei (andreimatei1@gmail.com)
+- RFC PR: 16000
+- Cockroach Issue: 13376, 10511
+
+
+# Summary
+
+The current split between the `client.Txn` and the `TxnCoordSender` doesn’t make
+sense any more. The current distinction between them is historical and has
+already led to hacks around the `Sender` interface through which the layers
+currently communicate. Also, the current split stands in the way of DistSQL,
+which needs access to some of the `TxnCoordSender` functionality but doesn’t
+actually send requests through it.
+
+This RFC is about agreeing that the current split is useless and inconvenient,
```

Comment:
Usually difficulty is solved through further splitting rather than merging as this RFC is suggesting. is it possible for you to consider further splitting things apart and understanding the components better before merging them later if you need to?